pitch.md

```diff
@@ -8,16 +8,101 @@ _Who are you helping, with your language? What is that person's
 experience like now? How would their experience be better if you could help 
 them?_
 
-The target audience is musicians of all levels. Currently, it is fairly difficult
-to improve one's sight reading bec
+The target audience is musicians of all levels. Musicians' current experience
+with sight reading is usually difficult because it is hard to find the right
+level of music to get the most out of a piece of music. I think with this project,
+their experience can improve by giving them the ability to find the right kind
+of music to sight read.
```

Comment:
Would this more be a language that helps them find the right kind of music to sight read (as in "Look at this composer for music for you" or to actually create the right type of music to sight read (the output is sheet music)? You mentioned that creating sheet music is a difficult problem but I imagine finding the right kind of music is also difficult.